packages/appeal-reply-service-api/src/models/reply.js

```diff
@@ -0,0 +1,92 @@
+const replyDocument = {
```

Comment:
This is the Appeals Service API spec - should be updated

Reply:
Done.